src/interface/sparse_pack.hpp

```diff
@@ -160,36 +160,54 @@ class SparsePack : public SparsePackBase {
   }
 
   // Host Bound overloads
-  KOKKOS_INLINE_FUNCTION int GetLowerBoundHost(const int b) const {
+  inline int GetLowerBoundHost(const int b) const {
```

Comment:
I don't remember why these were `KOKKOS_INLINE_FUNCTION`s in the first place (maybe just copy and paste?). That being said, I don't think the `inline` is necessary since class methods defined in the class definition are implicitly `inline` already.

Reply:
Ok I'll remove it

Reply:
done